src/System.Private.ServiceModel/tools/PRService/pr.ashx

```diff
@@ -243,11 +248,25 @@ public class PullRequestHandler : IHttpHandler
             success = RunGitCommands(new string[] { "branch --list" }, gitRepoPath, branchesString);
         }
 
-        executionResult.Append(branchesString);
+        executionResult.Append(branchesString.ToString());
 
         if (success)
         {
-            branches = branchesString.ToString().Split(new char[] { '\n', '\r', ' ', '*' }, StringSplitOptions.RemoveEmptyEntries);
+            tempBranches = branchesString.ToString().Split(new char[] { '\n', '\r' }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Reviewers: this is not going to win any coding competitions, but I figured that keeping this clean looking will help the next poor person (i.e., me) who looks at this